contracts/stake/OldTokens/StakedPSYS.sol

```diff
@@ -1,41 +0,0 @@
-// // SPDX-License-Identifier: agpl-3.0
```

Comment:
You can delete the "OldTokens" folder. I left them there to use as a reference for the StakedPSYS one.